packages/fxa-auth-server/lib/senders/emails/templates/fraudulentAccountDeletion/en.ftl

```diff
@@ -0,0 +1,8 @@
+fraudulentAccountDeletion-subject = Your { -brand-firefox } account was deleted
```

Comment:
The strings in this file should use the term for "Firefox account", not combine the term for Firefox and "account".

```suggestion
fraudulentAccountDeletion-subject = Your { -product-firefox-account } was deleted
```